setup.py

```diff
@@ -165,6 +113,16 @@ def run(self):
     'clean2': Clean,
 }
 
+
+# How the build process finds the system libs:
+#
+# 1. if BORG_{LIBXXX,OPENSSL}_PREFIX is set, it will use headers and libs from there.
+# 2. if not and pkg-config can locate the lib, the lib located by
+#    pkg-config will be used. We use the pkg-config tool via the pkgconfig
+#    python package, which must be installed before invoking setup.py.
+#    if pkgconfig is not installed, this step is skipped.
+# 3. otherwise raise a fatal error.
+
```

Comment:
Maybe we should move this quite a bit upwards, because if people run into issues with `setup.py`, they likely should read this first.

Reply:
OK. Moved it higher up after imports.